### PR TITLE
feat: add show password button for password input

### PR DESCRIPTION
**Pull request by G3root**
# Description 📣

[Screencast 2023-09-13 20:09:23.webm](https://github.com/Infisical/infisical/assets/84864519/0089ec8b-685d-4393-b56d-3e2bbe01fca0)

## Type ✨

- [x] New feature

- [x] I have read the [contributing guide](https://infisical.com/docs/contributing/overview), agreed and acknowledged the [code of conduct](https://infisical.com/docs/contributing/code-of-conduct). 📝

### CODE REVIEW

File: frontend/src/components/v2/Input/Input.tsx [hunk 1]
```diff
@@ -116,3 +120,30 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
...
+  (props, ref): JSX.Element => {
+    const [showPassword, setShowPassword] = useToggle(false);
+
+    return (
+      <Input
+        {...props}
+        ref={ref}
+        type={showPassword ? "text" : "password"}
+        rightIcon={
+          <button
```
Comment (akhilmhdh) [inline frontend/src/components/v2/Input/Input.tsx:136]: Can you please change this to IconButton component

File: frontend/src/views/Login/components/InitialStep/InitialStep.tsx [hunk 2]
```diff
@@ -2,222 +2,228 @@ import { FormEvent, useState } from "react";
...
+        <Button
+          type="submit"
+          size="sm"
+          isFullWidth
+          className="h-11"
+          colorSchema="primary"
+          variant="solid"
+          isLoading={isLoading}
+        >
+          {" "}
```
Comment (akhilmhdh) [inline frontend/src/views/Login/components/InitialStep/InitialStep.tsx:202]: Remove this please

**Review by akhilmhdh — CHANGES_REQUESTED** [review]

File: frontend/src/components/v2/Input/Input.tsx (hunk 1 above)
Comment (G3root) [reply frontend/src/components/v2/Input/Input.tsx]: Done 👍🏼

File: frontend/src/views/Login/components/InitialStep/InitialStep.tsx (hunk 2 above)
Comment (G3root) [reply frontend/src/views/Login/components/InitialStep/InitialStep.tsx]: Done 👍🏼

**Review by akhilmhdh — APPROVED** [review]

**Comment by maidul98** [comment]
Stale PR, closing for now

**Pull request closed without merging**